### PR TITLE
Add Wirex

### PR DESCRIPTION
**Pull request by kmpoppe**
Global alexa rank: 37,054

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -392,6 +392,13 @@ websites:
       software: Yes
       doc: https://support.uphold.com/hc/en-us/articles/360024042471
 
+    - name: wirex
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion
    - name: Wirex
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — DISMISSED** [review]

**Review by jtagcat — DISMISSED** [review]

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -392,6 +392,13 @@ websites:
       software: Yes
       doc: https://support.uphold.com/hc/en-us/articles/360024042471
 
+    - name: Wirex
+      url: https://wirexapp.com/
+      img: wirex.png
+      tfa: Yes
+      software: Yes
+      doc: https://wirexapp.com/help/article/how-to-enable-enhanced-security-2fa-for-my-wirex-account-0021
```
Comment (dsoegijono) [inline _data/cryptocurrencies.yml]: Do we want to use short URLs for doc? https://wirexapp.com/hc/articles/207160259

Comment (kmpoppe) [inline _data/cryptocurrencies.yml]:
```suggestion
      doc: https://wirexapp.com/hc/articles/207160259
```

Comment (kmpoppe) [reply _data/cryptocurrencies.yml]: @dsoegijono yes, we want short urls ;-)

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**